pyproject.toml

```diff
@@ -17,6 +17,7 @@ plotext = "^5.0.2"
 matplotlib = "^3.5.2"
 pandas = "^1.4.3"
 openai = "^0.23.0"
+kenlm = {version = "^0.2.0", optional = true}
```

Comment:
we don't want to force `kenlm` as a dependency---only install it if people need it